pkg/kp/kv.go

```diff
@@ -9,10 +9,17 @@ import (
 	"time"
 
 	"github.com/hashicorp/consul/api"
+	"gopkg.in/alecthomas/kingpin.v1"
+
 	"github.com/square/p2/pkg/pods"
 	"github.com/square/p2/pkg/util"
 )
 
+var showConsulErrors = kingpin.Flag(
```

Comment:
This is a pretty common use case for [gflags](https://gflags.github.io/gflags/) which got significantly simplified to become the standard [`flag`](http://golang.org/pkg/flag/) package.  I would have used `flag` if other p2 binaries hadn't already started using kingpin, and the two packages don't seem to get along well together.